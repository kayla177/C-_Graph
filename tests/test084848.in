ENTITY UYWJLAU8 name_UYWJ author
ENTITY YTR51T83 name_YTR5 conference
ENTITY LTI8YSW6 name_LTI8 institution
ENTITY 31NQ1WWF name_31NQ conference
ENTITY KLFDWJZ2 name_KLFD journal
ENTITY KGDK0ORJ name_KGDK institution
ENTITY PT1J9W7M name_PT1J journal
ENTITY 534BZ6ZQ name_534B paper
ENTITY GFR957Q8 name_GFR9 conference
ENTITY U5E3IXCH name_U5E3 conference
RELATIONSHIP LTI8YSW6 collaborated_with KLFDWJZ2 0.9
RELATIONSHIP GFR957Q8 authored_by LTI8YSW6 9.3
RELATIONSHIP LTI8YSW6 published_in KGDK0ORJ 9.7
RELATIONSHIP GFR957Q8 presented_at KLFDWJZ2 8.9
RELATIONSHIP KLFDWJZ2 published_in KGDK0ORJ 9.1
RELATIONSHIP YTR51T83 published_in 534BZ6ZQ 9.1
RELATIONSHIP UYWJLAU8 collaborated_with GFR957Q8 2.7
RELATIONSHIP 534BZ6ZQ cited_by KGDK0ORJ 6.3
RELATIONSHIP PT1J9W7M cited_by YTR51T83 3.7
RELATIONSHIP KLFDWJZ2 authored_by YTR51T83 7.8
RELATIONSHIP LTI8YSW6 collaborated_with U5E3IXCH 9.6
RELATIONSHIP PT1J9W7M affiliated_with GFR957Q8 7.6
RELATIONSHIP UYWJLAU8 published_in 31NQ1WWF 6.2
RELATIONSHIP LTI8YSW6 presented_at 31NQ1WWF 7.3
RELATIONSHIP 31NQ1WWF presented_at YTR51T83 6.6
PATH 31NQ1WWF KGDK0ORJ
HIGHEST
HIGHEST
PRINT LTI8YSW6
DELETE KGDK0ORJ
PATH YTR51T83 534BZ6ZQ
DELETE LTI8YSW6
PATH 31NQ1WWF 534BZ6ZQ
DELETE KLFDWJZ2
HIGHEST
EXIT
