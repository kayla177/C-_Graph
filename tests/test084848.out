success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
31NQ1WWF LTI8YSW6 KGDK0ORJ 17
KLFDWJZ2 U5E3IXCH 28.4
KLFDWJZ2 U5E3IXCH 28.4
KLFDWJZ2 GFR957Q8 KGDK0ORJ U5E3IXCH 31NQ1WWF
success
YTR51T83 534BZ6ZQ 9.1
success
31NQ1WWF YTR51T83 534BZ6ZQ 15.7
success
UYWJLAU8 534BZ6ZQ 21.9
